sentinelsat/sentinel.py

```diff
@@ -675,8 +675,8 @@ def check_files(self, paths=None, ids=None, directory=None, delete=False):
         paths = paths or []
         ids = ids or []
 
-        def name_from_path(path):
-            return splitext(basename(path))[0]
+        def name_from_path(base_path):
```

Comment:
No, please, `path` is absolutely legal to use here. No one in their right might should use `from os import path`. 😄

Reply:
I renamed it not because of os.path (which is imported in another way), but because we use "path" as a variable name in the outer function.

Reply:
Since this is a very narrowly scoped variable, the name does not matter too much. I don't think `base_path` makes too much sense as the variable name though (it's not used as a base path for anything?). Since the function parameter will not affect the variable in the outer scope and vice versa, I would keep it as it is.

Reply:
I use here the same reasoning as with geometry, that using different names in inner/outer scopes may save some trouble in future(or it may not). Anyway, right now this is definitely not a problem, so I'm not opposed to leaving it as it is.